plugin/src/main/groovy/com/github/kright/androidscalasupport/MainDexOverwrite.groovy

```diff
@@ -0,0 +1,24 @@
+package com.github.kright.androidscalasupport
+
+/**
+ * Created by lgor on 01.03.2017.
+ */
```

Comment:
`MainDexOverwrite` is a strange name, consider `MainDexRules` or `MainDexOptions`